pvlib/modelchain.py

```diff
@@ -644,7 +644,9 @@ def orientation_strategy(self, strategy):
             strategy = None
 
         if strategy is not None:
-            self.system.surface_tilt, self.system.surface_azimuth = \
+            # TODO: this is probably not what we want to do here
```

Comment:
Not sure what to think about this orientation strategy business. `self.system` might have arrays with different mounts.  Error if `len(self.system.arrays) > 1`, and maybe also if the array's mount isn't a `FixedMount`?

Reply:
remove it? https://github.com/pvlib/pvlib-python/issues/1028